README.md

```diff
@@ -356,6 +356,96 @@ countItems := response.CountItems
 count := response.Count
 ```
 
+## Importing and Exporting Data
+
+### Importing Data
+
+If you have large amounts of data, it is more efficient to import it to Pilosa instead of several `SetBit` queries. This library supports importing bits of a frame. The target frame must have been created before hand.
```

Comment:
If you have large amounts of data, it is more efficient to import it into Pilosa instead of using multiple `SetBit` queries. This library supports importing bits into an existing frame.